test/FunctionsV2/AzureStorageModifiedDurabilityProviderFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
+{
+    internal class AzureStorageModifiedDurabilityProviderFactory : AzureStorageDurabilityProviderFactory
```

Comment:
This name is very vague.

I would go for something like `AzureStorageShortenedTimerDurabilityProviderFactory`.